tox.ini

```diff
@@ -7,7 +7,7 @@
 twisted = 1
 
 [tox]
-envlist = {py27,pypy27,py36}{-coverage,}
+envlist = codechecks,py27,py36,pypy27
```

Comment:
Why do we remove the `-coverage` variants from the list here?

Reply:
Does [the commit message for that change] answer your question?

Reply:
oh, this is just the default list (and you can still run "py27-coverage" if you want) ..?

Reply:
Correct.